ckan/config/environment.py

```diff
@@ -96,7 +90,29 @@ def __getattr__(self, name):
                 msg = 'Helper function `%s` could not be found\n ' \
                       '(are you missing an extension?)' % name
                 self.log.critical(msg)
-            return self.null_function
+            return _null_function
+
+
+def _null_function(*args, **kw):
+    ''' This function is returned if no helper is found. The idea is
+    to try to allow templates to be rendered even if helpers are
+    missing.  Returning the empty string seems to work well.'''
+    return ''
```

Comment:
@brew I moved this out because there's no reason for it to be a `classmethod` (it never uses `cls`), and because as a normal function I can do the `is _null_function` check below